client/packages/system/src/Patient/hooks/usePatientModalStore.ts

```diff
@@ -31,9 +46,37 @@ export const usePatientModalStore = zustand<PatientModalState>(set => ({
       document: undefined,
       programType: undefined,
     })),
-  setCurrent: (current?: PatientModal) => set(state => ({ ...state, current })),
-  setDocument: (document?: PatientModalDocument) =>
-    set(state => ({ ...state, document })),
-  setProgramType: (programType?: string) =>
-    set(state => ({ ...state, programType })),
+  selectModal: (current?: PatientModal) =>
+    set(state => ({
+      ...state,
+      document: undefined,
+      programType: undefined,
+      current,
+    })),
+
+  setEditingModal: (
```

Comment:
:nit

the terms 'editing' and 'creation' are different tenses of the verbs. Myself, I'd use `setEditModal` and `setCreateModal`.. but if you want the `setEditingModal` version then I'd rename the other to `setCreatingModal`

Reply:
thanks! [139aad4](https://github.com/openmsupply/open-msupply/pull/528/commits/139aad4b272b85141e4c6c7907b1bf6f1d620043)